[163.com] HTTPConnectionPool(host='163.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9cb3597550>, 'Connection to 163.com timed out. (connect timeout=None)'))
[a-msedge.net] HTTPConnectionPool(host='a-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3660a90>: Failed to resolve 'a-msedge.net' ([Errno -5] No address associated with hostname)"))
[a2z.com] HTTPConnectionPool(host='a2z.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3664a10>: Failed to resolve 'a2z.com' ([Errno -5] No address associated with hostname)"))
[aaplimg.com] HTTPConnectionPool(host='aaplimg.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3666750>: Failed to resolve 'aaplimg.com' ([Errno -5] No address associated with hostname)"))
[adnxs.com] HTTPConnectionPool(host='adnxs.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3678450>: Failed to resolve 'adnxs.com' ([Errno -5] No address associated with hostname)"))
[aiv-cdn.net] HTTPConnectionPool(host='aiv-cdn.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb35974d0>: Failed to resolve 'aiv-cdn.net' ([Errno -5] No address associated with hostname)"))
[akadns.net] HTTPConnectionPool(host='akadns.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb359cfd0>: Failed to resolve 'akadns.net' ([Errno -5] No address associated with hostname)"))
[akamai.net] HTTPConnectionPool(host='akamai.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3664c10>: Failed to resolve 'akamai.net' ([Errno -5] No address associated with hostname)"))
[akamaiedge.net] HTTPConnectionPool(host='akamaiedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb3662410>: Failed to resolve 'akamaiedge.net' ([Errno -5] No address associated with hostname)"))
[apple-dns.net] HTTPConnectionPool(host='apple-dns.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb36311d0>: Failed to resolve 'apple-dns.net' ([Errno -5] No address associated with hostname)"))
[b-msedge.net] HTTPConnectionPool(host='b-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8a7e290>: Failed to resolve 'b-msedge.net' ([Errno -5] No address associated with hostname)"))
[cloudapp.net] HTTPConnectionPool(host='cloudapp.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8509d50>: Failed to resolve 'cloudapp.net' ([Errno -5] No address associated with hostname)"))
[cloudfront.net] HTTPConnectionPool(host='cloudfront.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb9790250>: Failed to resolve 'cloudfront.net' ([Errno -5] No address associated with hostname)"))
[csdn.net] HTTPSConnectionPool(host='www.csdn.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb8886310>, 'Connection to www.csdn.net timed out. (connect timeout=None)'))
[domaincontrol.com] HTTPConnectionPool(host='domaincontrol.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9cb91d9750>: Failed to establish a new connection: [Errno 111] Connection refused'))
[dzeninfra.ru] HTTPConnectionPool(host='dzeninfra.ru', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8efbcd0>: Failed to resolve 'dzeninfra.ru' ([Errno -5] No address associated with hostname)"))
[edgekey.net] HTTPConnectionPool(host='edgekey.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8ee3e90>: Failed to resolve 'edgekey.net' ([Errno -5] No address associated with hostname)"))
[edgesuite.net] HTTPConnectionPool(host='edgesuite.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8ee1190>: Failed to resolve 'edgesuite.net' ([Errno -5] No address associated with hostname)"))
[googleusercontent.com] HTTPConnectionPool(host='googleusercontent.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb85feb50>: Failed to resolve 'googleusercontent.com' ([Errno -5] No address associated with hostname)"))
[gtld-servers.net] HTTPConnectionPool(host='gtld-servers.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8515590>: Failed to resolve 'gtld-servers.net' ([Errno -5] No address associated with hostname)"))
[gvt1.com] HTTPConnectionPool(host='gvt1.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb9733a50>: Failed to resolve 'gvt1.com' ([Errno -5] No address associated with hostname)"))
[gvt2.com] HTTPConnectionPool(host='gvt2.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb9470910>: Failed to resolve 'gvt2.com' ([Errno -5] No address associated with hostname)"))
[health.mil] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[l-msedge.net] HTTPConnectionPool(host='l-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb865ea90>: Failed to resolve 'l-msedge.net' ([Errno -5] No address associated with hostname)"))
[macromedia.com] HTTPConnectionPool(host='macromedia.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9cb89bea50>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[mangosip.ru] HTTPConnectionPool(host='mangosip.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9cb94663d0>, 'Connection to mangosip.ru timed out. (connect timeout=None)'))
[microsoftonline.com] HTTPConnectionPool(host='microsoftonline.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb96e2090>: Failed to resolve 'microsoftonline.com' ([Errno -5] No address associated with hostname)"))
[miit.gov.cn] HTTPConnectionPool(host='miit.gov.cn', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb85f8410>: Failed to resolve 'miit.gov.cn' ([Errno -5] No address associated with hostname)"))
[msedge.net] HTTPConnectionPool(host='msedge.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9cba71b510>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[name-services.com] HTTPConnectionPool(host='name-services.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8d31710>: Failed to resolve 'name-services.com' ([Errno -5] No address associated with hostname)"))
[netflix.net] HTTPConnectionPool(host='netflix.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9cb8831790>, 'Connection to netflix.net timed out. (connect timeout=None)'))
[nflxso.net] HTTPConnectionPool(host='nflxso.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb938b510>: Failed to resolve 'nflxso.net' ([Errno -5] No address associated with hostname)"))
[nih.gov] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[o365filtering.com] HTTPConnectionPool(host='o365filtering.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8689b90>: Failed to resolve 'o365filtering.com' ([Errno -5] No address associated with hostname)"))
[office.net] HTTPConnectionPool(host='office.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb9693f10>: Failed to resolve 'office.net' ([Errno -5] No address associated with hostname)"))
[omtrdc.net] HTTPConnectionPool(host='omtrdc.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8677ad0>: Failed to resolve 'omtrdc.net' ([Errno -5] No address associated with hostname)"))
[registrar-servers.com] HTTPConnectionPool(host='registrar-servers.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8695750>: Failed to resolve 'registrar-servers.com' ([Errno -5] No address associated with hostname)"))
[ruidongcloud.com] HTTPConnectionPool(host='ruidongcloud.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8517450>: Failed to resolve 'ruidongcloud.com' ([Errno -5] No address associated with hostname)"))
[s-msedge.net] HTTPConnectionPool(host='s-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8515690>: Failed to resolve 's-msedge.net' ([Errno -5] No address associated with hostname)"))
[spo-msedge.net] HTTPConnectionPool(host='spo-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8f20850>: Failed to resolve 'spo-msedge.net' ([Errno -5] No address associated with hostname)"))
[t-msedge.net] HTTPConnectionPool(host='t-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8c16450>: Failed to resolve 't-msedge.net' ([Errno -5] No address associated with hostname)"))
[tiktokcdn.com] HTTPConnectionPool(host='tiktokcdn.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8b0a790>: Failed to resolve 'tiktokcdn.com' ([Errno -5] No address associated with hostname)"))
[trafficmanager.net] HTTPConnectionPool(host='trafficmanager.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8a73550>: Failed to resolve 'trafficmanager.net' ([Errno -5] No address associated with hostname)"))
[trbcdn.net] ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
[twimg.com] HTTPConnectionPool(host='twimg.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8a598d0>: Failed to resolve 'twimg.com' ([Errno -5] No address associated with hostname)"))
[wac-msedge.net] HTTPConnectionPool(host='wac-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb83c2f90>: Failed to resolve 'wac-msedge.net' ([Errno -5] No address associated with hostname)"))
[windowsupdate.com] HTTPConnectionPool(host='windowsupdate.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb8f96790>: Failed to resolve 'windowsupdate.com' ([Errno -5] No address associated with hostname)"))
[wsdvs.com] HTTPConnectionPool(host='wsdvs.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f9cb93fd050>: Failed to resolve 'wsdvs.com' ([Errno -5] No address associated with hostname)"))
